app/controllers/concerns/foreman/controller/provisioning_templates.rb

```diff
@@ -42,4 +43,21 @@ def process_template_kind
   def type_name_singular
     @type_name_singular ||= resource_class.to_s.underscore
   end
+
+  def global_registration_vars
+    permitted = Foreman::Plugin.all
+                               .map(&:allowed_registration_vars)
+                               .flatten.compact.uniq
+    organization = params['organization_id'].present? ? Organization.authorized.find(params['organization_id']) : Organization.current
```

Comment:
I think a permission needs to be passed to `authorized`. E.g. `authorized(:view_organizations, Organization)` (the second param may not be necessary)

the same applies below to Location